app/scripts/views/museum/museum-controller.js

```diff
@@ -228,6 +231,17 @@
             });
         }
 
+        function setACSArea(areaMeters) {
+            var areaMiles = Math.abs(areaMeters) * 0.000621371 * 0.000621371;
```

Comment:
Yup, converting from area to area so you have to multiply by the conversion factor for each dimension.